SoCCo/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Empty file
+
+from algorithms.climate import computeRF
```

Comment:
Each of these lines should have a SoCCo in front of them to make sure it is explicit which algorithms package we are using here.